### PR TITLE
Added verified_contents.json generator.

### PR DESCRIPTION
**Pull request by boocmp**
Resolves https://github.com/brave/brave-browser/issues/42274

### CODE REVIEW

File: lib/util.js [hunk 1]
```diff
@@ -63,6 +64,30 @@ const fetchTextFromURL = (listURL) => {
   return p
 }
 
+const generateComponentMetadata = (
+  inputDir,
+  publisherProofKey,
+  publisherProofKeyAlt
```
Comment (diracdeltas) [inline lib/util.js:70]: are we using the existing crx signing keys for this?

for production, is this going to be called in CI?

Comment (boocmp) [reply lib/util.js:70]: I'm not sure. I am inclined to generate a new one for this purpose.

Comment (boocmp) [reply lib/util.js]: `Components Verified Contents PEM` key in 1Password

**Review by antonok-edm — APPROVED** [review]

**Review by diracdeltas — APPROVED** [review]
might be good to have tests with a fake private key but lgtm

**Review by linhkikuchi — APPROVED** [review]

**Pull request merged**